### ISSUE TITLE
[Bug] JungleSkeletons cause Exception

### ISSUE THREAD

**Issue by quiqueck** [paulevsGitch/BetterNether#452]
### What happened?

Jungle Skeletons crash my game on any version of better nether that uses bclib.

Jungle Skeletons move very fast and crash the game when they deal damage in melee, ranged attacks with a bow seem to not have this effect.

Jungle Skeletons work as expected on Better-nether-5.1.3

### BetterNether

5.3.13

### BCLib

0.5.5

### Fabric API

0.44.0+1.17

### Fabric Loader

-

### Minecraft

1.17.1

### Relevant log output

```shell
[18:04:34] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Ticking entity
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:895) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:831) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1132.method_3748(class_1132.java:96) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:697) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
Caused by: java.lang.IllegalArgumentException: Can't find attribute minecraft:generic.attack_damage
	at net.minecraft.class_5132.method_26865(class_5132.java:22) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_5132.method_26862(class_5132.java:28) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_5131.method_26852(class_5131.java:62) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1309.method_26825(class_1309.java:1860) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1308.method_6121(class_1308.java:1388) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1366.method_6288(class_1366.java:142) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1366.method_6268(class_1366.java:134) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_4135.method_6268(class_4135.java:55) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179) ~[?:?]
	at java.util.Iterator.forEachRemaining(Iterator.java:133) ~[?:?]
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845) ~[?:?]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?]
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
	at net.minecraft.class_1355.method_6275(class_1355.java:88) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1308.method_6023(class_1308.java:733) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1309.method_6007(class_1309.java:2601) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1308.method_6007(class_1308.java:539) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1588.method_6007(class_1588.java:43) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1547.method_6007(class_1547.java:126) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at paulevs.betternether.entity.EntityJungleSkeleton.method_6007(EntityJungleSkeleton.java:32) ~[better-nether-5.3.13.jar:?]
	at net.minecraft.class_1309.method_5773(class_1309.java:2346) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1308.method_5773(class_1308.java:339) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1613.method_5773(class_1613.java:71) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_3218.method_18762(class_3218.java:655) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_1937.method_18472(class_1937.java:483) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_3218.method_31420(class_3218.java:417) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_5574.method_31791(class_5574.java:54) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.class_3218.method_18765(class_3218.java:392) ~[intermediary-minecraft-1.17.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:891) ~[intermediary-minecraft-1.17.1-client.jar:?]
	... 5 more
[18:04:34] [Server thread/ERROR]: This crash report has been saved to: C:\Users\User\Desktop\MultiMC\instances\1.17.1\.minecraft\crash-reports\crash-2021-12-06_18.04.34-server.txt
[18:04:34] [Server thread/INFO]: Stopping server
[18:04:34] [Server thread/INFO]: Saving players
[18:04:34] [Server thread/INFO]: VortigonMusic lost connection: Disconnected
[18:04:34] [Server thread/INFO]: VortigonMusic left the game
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 12/6/21, 6:04 PM
Description: Ticking entity

java.lang.IllegalArgumentException: Can't find attribute minecraft:generic.attack_damage
	at net.minecraft.class_5132.method_26865(class_5132.java:22)
	at net.minecraft.class_5132.method_26862(class_5132.java:28)
	at net.minecraft.class_5131.method_26852(class_5131.java:62)
	at net.minecraft.class_1309.method_26825(class_1309.java:1860)
	at net.minecraft.class_1308.method_6121(class_1308.java:1388)
	at net.minecraft.class_1366.method_6288(class_1366.java:142)
	at net.minecraft.class_1366.method_6268(class_1366.java:134)
	at net.minecraft.class_4135.method_6268(class_4135.java:55)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at net.minecraft.class_1355.method_6275(class_1355.java:88)
	at net.minecraft.class_1308.method_6023(class_1308.java:733)
	at net.minecraft.class_1309.method_6007(class_1309.java:2601)
	at net.minecraft.class_1308.method_6007(class_1308.java:539)
	at net.minecraft.class_1588.method_6007(class_1588.java:43)
	at net.minecraft.class_1547.method_6007(class_1547.java:126)
	at paulevs.betternether.entity.EntityJungleSkeleton.method_6007(EntityJungleSkeleton.java:32)
	at net.minecraft.class_1309.method_5773(class_1309.java:2346)
	at net.minecraft.class_1308.method_5773(class_1308.java:339)
	at net.minecraft.class_1613.method_5773(class_1613.java:71)
	at net.minecraft.class_3218.method_18762(class_3218.java:655)
	at net.minecraft.class_1937.method_18472(class_1937.java:483)
	at net.minecraft.class_3218.method_31420(class_3218.java:417)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_3218.method_18765(class_3218.java:392)
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:891)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:831)
	at net.minecraft.class_1132.method_3748(class_1132.java:96)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:697)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)
	at java.base/java.lang.Thread.run(Thread.java:833)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_5132.method_26865(class_5132.java:22)
	at net.minecraft.class_5132.method_26862(class_5132.java:28)
	at net.minecraft.class_5131.method_26852(class_5131.java:62)
	at net.minecraft.class_1309.method_26825(class_1309.java:1860)
	at net.minecraft.class_1308.method_6121(class_1308.java:1388)
	at net.minecraft.class_1366.method_6288(class_1366.java:142)
	at net.minecraft.class_1366.method_6268(class_1366.java:134)
	at net.minecraft.class_4135.method_6268(class_4135.java:55)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at net.minecraft.class_1355.method_6275(class_1355.java:88)
	at net.minecraft.class_1308.method_6023(class_1308.java:733)
	at net.minecraft.class_1309.method_6007(class_1309.java:2601)
	at net.minecraft.class_1308.method_6007(class_1308.java:539)
	at net.minecraft.class_1588.method_6007(class_1588.java:43)
	at net.minecraft.class_1547.method_6007(class_1547.java:126)
	at paulevs.betternether.entity.EntityJungleSkeleton.method_6007(EntityJungleSkeleton.java:32)
	at net.minecraft.class_1309.method_5773(class_1309.java:2346)
	at net.minecraft.class_1308.method_5773(class_1308.java:339)
	at net.minecraft.class_1613.method_5773(class_1613.java:71)
	at net.minecraft.class_3218.method_18762(class_3218.java:655)

-- Entity being ticked --
Details:
	Entity Type: betternether:jungle_skeleton (paulevs.betternether.entity.EntityJungleSkeleton)
	Entity ID: 1640
	Entity Name: Jungle Skeleton
	Entity's Exact location: -42.52, 70.23, -49.30
	Entity's Block location: World: (-43,70,-50), Section: (at 5,6,14 in -3,4,-4; chunk contains blocks -48,0,-64 to -33,255,-49), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Entity's Momentum: 0.32, -0.38, 0.00
	Entity's Passengers: []
	Entity's Vehicle: null
Stacktrace:
	at net.minecraft.class_1937.method_18472(class_1937.java:483)
	at net.minecraft.class_3218.method_31420(class_3218.java:417)
	at net.minecraft.class_5574.method_31791(class_5574.java:54)
	at net.minecraft.class_3218.method_18765(class_3218.java:392)

-- Affected level --
Details:
	All players: 1 total; [class_3222['VortigonMusic'/574, l='ServerLevel[New World]', x=-41.88, y=70.00, z=-50.32]]
	Chunk stats: 3539
	Level dimension: minecraft:overworld
	Level spawn location: World: (-48,70,-48), Section: (at 0,6,0 in -3,4,-3; chunk contains blocks -48,0,-48 to -33,255,-33), Region: (-1,-1; contains chunks -32,-32 to -1,-1, blocks -512,0,-512 to -1,255,-1)
	Level time: 329 game time, 329 day time
	Level name: New World
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: true
	Level weather: Rain time: 175805 (now: false), thunder time: 26471 (now: false)
	Known server brands: fabric
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
Stacktrace:
	at net.minecraft.server.MinecraftServer.method_3813(MinecraftServer.java:891)
	at net.minecraft.server.MinecraftServer.method_3748(MinecraftServer.java:831)
	at net.minecraft.class_1132.method_3748(class_1132.java:96)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:697)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:270)
	at java.base/java.lang.Thread.run(Thread.java:833)

-- System Details --
Details:
	Minecraft Version: 1.17.1
	Minecraft Version ID: 1.17.1
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode, sharing), Oracle Corporation
	Memory: 3074422784 bytes (2931 MiB) / 3963617280 bytes (3780 MiB) up to 6316621824 bytes (6024 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 5600X 6-Core Processor             
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 0
	Microarchitecture: Zen 3
	Frequency (GHz): 3.69
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: NVIDIA GeForce RTX 3070 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2482
	Graphics card #0 versionInfo: DriverVersion=30.0.14.9649
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 16384.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 37578.43
	Virtual memory used (MB): 23986.54
	Swap memory total (MB): 4864.00
	Swap memory used (MB): 36.22
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx6024m
	Fabric Mods: 
		bclib: BCLib 0.5.5
		betternether: Better Nether 5.3.13
		fabric: Fabric API 0.44.0+1.17
		fabric-api-base: Fabric API Base 0.4.0+cf39a74318
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.4.0+cb9053d118
		fabric-biome-api-v1: Fabric Biome API (v1) 3.2.2+cf39a74318
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.6+cf39a74318
		fabric-command-api-v1: Fabric Command API (v1) 1.1.4+cf39a74318
		fabric-commands-v0: Fabric Commands (v0) 0.2.3+cf39a74318
		fabric-containers-v0: Fabric Containers (v0) 0.1.13+cf39a74318
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.4.1+3447790d18
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.6+cf39a74318
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.13+cf39a74318
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.1+377137cc18
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.12+e99fbe1218
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.4+cf39a74318
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.8+cf39a74318
		fabric-item-api-v1: Fabric Item API (v1) 1.3.0+6617390918
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.2+cf39a74318
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.6+2a2bb57318
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.4+cf39a74318
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.6+0392f3a618
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.5+cf39a74318
		fabric-mining-level-api-v1: Fabric Mining Level API (v1) 1.0.3+cf39a74318
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.6+cf39a74318
		fabric-models-v0: Fabric Models (v0) 0.3.1+cf39a74318
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.14+cf39a74318
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.12+cf39a74318
		fabric-networking-v0: Fabric Networking (v0) 0.3.3+cf39a74318
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.11.0+f907116918
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.7+cf39a74318
		fabric-particles-v1: Fabric Particles (v1) 0.2.5+cf39a74318
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.13+cf39a74318
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.5+cf39a74318
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.9+cf39a74318
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.6+cf39a74318
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.6+cf39a74318
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.2.1+cf39a74318
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.7+cf39a74318
		fabric-rendering-v1: Fabric Rendering (v1) 1.10.1+377137cc18
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.10+f09604ce18
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.5+cf39a74318
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.9+cf39a74318
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.15+cf39a74318
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.2.3+cf39a74318
		fabric-textures-v0: Fabric Textures (v0) 1.0.7+cf39a74318
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.3.2+cf39a74318
		fabric-transfer-api-v1: Fabric Transfer API (v1) 1.5.3+cf39a74318
		fabricloader: Fabric Loader 0.12.8
		java: Java HotSpot(TM) 64-Bit Server VM 17
		minecraft: Minecraft 1.17.1
```

### Other Mods

```shell
- bclib 0.5.5
	- betternether 5.3.13
	- fabric 0.44.0+1.17
	- fabric-api-base 0.4.0+cf39a74318 via fabric
	- fabric-api-lookup-api-v1 1.4.0+cb9053d118 via fabric
	- fabric-biome-api-v1 3.2.2+cf39a74318 via fabric
	- fabric-blockrenderlayer-v1 1.1.6+cf39a74318 via fabric
	- fabric-command-api-v1 1.1.4+cf39a74318 via fabric
	- fabric-commands-v0 0.2.3+cf39a74318 via fabric
	- fabric-containers-v0 0.1.13+cf39a74318 via fabric
	- fabric-content-registries-v0 0.4.1+3447790d18 via fabric
	- fabric-crash-report-info-v1 0.1.6+cf39a74318 via fabric
	- fabric-dimensions-v1 2.0.13+cf39a74318 via fabric
	- fabric-entity-events-v1 1.4.1+377137cc18 via fabric
	- fabric-events-interaction-v0 0.4.12+e99fbe1218 via fabric
	- fabric-events-lifecycle-v0 0.2.4+cf39a74318 via fabric
	- fabric-game-rule-api-v1 1.0.8+cf39a74318 via fabric
	- fabric-item-api-v1 1.3.0+6617390918 via fabric
	- fabric-item-groups-v0 0.3.2+cf39a74318 via fabric
	- fabric-key-binding-api-v1 1.0.6+2a2bb57318 via fabric
	- fabric-keybindings-v0 0.2.4+cf39a74318 via fabric
	- fabric-lifecycle-events-v1 1.4.6+0392f3a618 via fabric
	- fabric-loot-tables-v1 1.0.5+cf39a74318 via fabric
	- fabric-mining-level-api-v1 1.0.3+cf39a74318 via fabric
	- fabric-mining-levels-v0 0.1.6+cf39a74318 via fabric
	- fabric-models-v0 0.3.1+cf39a74318 via fabric
	- fabric-networking-api-v1 1.0.14+cf39a74318 via fabric
	- fabric-networking-blockentity-v0 0.2.12+cf39a74318 via fabric
	- fabric-networking-v0 0.3.3+cf39a74318 via fabric
	- fabric-object-builder-api-v1 1.11.0+f907116918 via fabric
	- fabric-object-builders-v0 0.7.7+cf39a74318 via fabric
	- fabric-particles-v1 0.2.5+cf39a74318 via fabric
	- fabric-registry-sync-v0 0.7.13+cf39a74318 via fabric
	- fabric-renderer-api-v1 0.4.5+cf39a74318 via fabric
	- fabric-renderer-indigo 0.4.9+cf39a74318 via fabric
	- fabric-renderer-registries-v1 3.2.6+cf39a74318 via fabric
	- fabric-rendering-data-attachment-v1 0.1.6+cf39a74318 via fabric
	- fabric-rendering-fluids-v1 0.2.1+cf39a74318 via fabric
	- fabric-rendering-v0 1.1.7+cf39a74318 via fabric
	- fabric-rendering-v1 1.10.1+377137cc18 via fabric
	- fabric-resource-loader-v0 0.4.10+f09604ce18 via fabric
	- fabric-screen-api-v1 1.0.5+cf39a74318 via fabric
	- fabric-screen-handler-api-v1 1.1.9+cf39a74318 via fabric
	- fabric-structure-api-v1 1.1.15+cf39a74318 via fabric
	- fabric-tag-extensions-v0 1.2.3+cf39a74318 via fabric
	- fabric-textures-v0 1.0.7+cf39a74318 via fabric
	- fabric-tool-attribute-api-v1 1.3.2+cf39a74318 via fabric
	- fabric-transfer-api-v1 1.5.3+cf39a74318 via fabric
	- fabricloader 0.12.8
	- java 17
	- minecraft 1.17.1
```

**Comment by Tatitooo** [comment]
This also happens in 1.17.1 using:
better-nether-5.3.12

**Comment by quiqueck** [comment]
Released with 6.0.2 and 5.3.14
